### ISSUE TITLE
a common label for a radiobuttons group

### ISSUE THREAD

**Issue by kranid** [pixelstrolch/accessible-pdf#8]
how can i set a common label for a radiobuttons group in PDF documents like using Legend<legend> in html. I'd like screenreader to announce this comon label when a user focuses on the  first radiobutton of this group.

**Comment by stebrech** [comment]
Hi kranid, this repository is used to manage the content of accessible-pdf.info. Please open an issue if you suggest an new article the documentation site should cover or to fix an existing one. It's not a useful place for asking specific questions. There are some places with an experienced audience, for example:

- Adobe Forum [Accessibility & Standards](https://forums.adobe.com/community/acrobat/accessibility_&_standards)
- [Slack group A11Y](web-a11y.slack.com), channel PDF
- LinkedIn Group [“About PDF/UA (accessibility)”](https://www.linkedin.com/groups/7470079/)

In my opinion screenreader should use the alt text of the radio buttons group. The name and the **Tooltip (alt text) is the same for each** radio button of the group. You define the value of each radio button with the "Radio Button Choice" within the Radio Button Properties.
